x-pack/plugins/observability/public/pages/slo_details/components/error_budget_chart_panel.tsx

```diff
@@ -27,6 +28,24 @@ export function ErrorBudgetChartPanel({ data, isLoading, slo }: Props) {
 
   const isSloFailed = slo.summary.status === 'DEGRADING' || slo.summary.status === 'VIOLATED';
 
+  let errorBudgetTimeRemainingFormatted;
+  if (slo.budgetingMethod === 'timeslices' && slo.timeWindow.type === 'calendarAligned') {
+    const totalSlices =
+      toMinutes(toDuration(slo.timeWindow.duration)) /
+      toMinutes(toDuration(slo.objective.timesliceWindow!));
+    const errorBudgetRemainingInMinute =
+      slo.summary.errorBudget.remaining * (slo.summary.errorBudget.initial * totalSlices);
+
+    if (errorBudgetRemainingInMinute > 0) {
+      if (errorBudgetRemainingInMinute / 60 >= 1) {
+        errorBudgetTimeRemainingFormatted = `${Math.trunc(errorBudgetRemainingInMinute / 60)}h`;
+      }
+      errorBudgetTimeRemainingFormatted += `${Math.trunc(errorBudgetRemainingInMinute % 60)}min`;
+    } else {
+      errorBudgetTimeRemainingFormatted = `0min`;
+    }
```

Comment:
Can we changed the format to `1h 35m` similar to how the new alert time table is formatted:

https://github.com/elastic/kibana/blob/b247e892be2da3307ad15ff3016571e1497ed0ab/x-pack/plugins/observability/public/components/burn_rate_rule_editor/alert_time_table.tsx#L23-L36

Also we could probably improve that formatter to handle situations where there are exactly 60 minutes and instead of printing out `1 h 0m` we print `1h`

Reply:
Going to reuse that 👍🏻